etc/profile

```diff
@@ -61,6 +61,13 @@ MINGW64)
   ACLOCAL_PATH="${MINGW_MOUNT_POINT}/share/aclocal:/usr/share/aclocal"
   MANPATH="${MINGW_MOUNT_POINT}/local/man:${MINGW_MOUNT_POINT}/share/man:${MANPATH}"
   ;;
+ARM64)
+  MINGW_MOUNT_POINT="${MINGW_PREFIX}"
```

Comment:
MINGW_PREFIX is empty on my arm64 test build 🤷🏼‍♂️ 

I saw some references to MINGW_PREFIX in various places and it mostly relies on `uname -m`. Since Bash will remain a 32-bit executable for now, it's probably best if we set `MINGW_MOUNT_POINT="/arm64" here.

Before setting `MINGW_MOUNT_POINT` to `/arm64`:

After: